tidy3d/components/base.py

```diff
@@ -212,7 +212,7 @@ def _special_characters_not_in_name(cls, values):
 
     def copy(self, deep: bool = True, **kwargs) -> Tidy3dBaseModel:
         """Copy a Tidy3dBaseModel.  With ``deep=True`` as default."""
-        kwargs.update(deep=deep)
+        # kwargs.update(deep=deep)
```

Comment:
intentional?

Reply:
no! was testing something and forgot to change it back. sorry about that.